services/api-server/src/simcore_service_api_server/utils/client_base.py

```diff
@@ -26,7 +29,8 @@ async def is_responsive(self) -> bool:
             resp = await self.client.get(self.health_check_path)
             resp.raise_for_status()
             return True
-        except (httpx.HTTPStatusError, httpx.RequestError):
+        except (httpx.HTTPStatusError, httpx.RequestError) as err:
+            log.debug("%s not responsive: %s", self.service_name, err)
```

Comment:
maybe add this as a warning? if you really want to see it in staging and production environments